meta-resin-common/classes/binary-compress.bbclass

```diff
@@ -0,0 +1,28 @@
+# e.g.  COMPRESS_BINARY = "path/bin1 path/bin2 path/bin3"
```

Comment:
I would propose to have them as FILES_ work. And entries in FILES are absolute but they are treated as relative paths in code.